amp_conf/htdocs/admin/libraries/BMO/DB_Helper.class.php

```diff
@@ -321,7 +321,7 @@ public function setConfig($key = null, $val = false, $id = "noid") {
 				$uuid = $this->insertBlob(null, $query[':val'], $query[':type']);
 			}
 			$query[':type'] = "blob";
-			$query[':val'] = $uuid;
+			$val = $uuid;
```

Comment:
You need to set val AND the query. The way you did it the truncated value is stored in the database and not the uuid

Reply:
Yes, that makes sense; in all my testing the code exited before hitting the statement execution but I can see where that might not always be the case.